src/Sarif/Writers/SarifLogger.cs

```diff
@@ -447,12 +447,16 @@ private void CaptureArtifact(ArtifactLocation fileLocation)
                 catch (ArgumentException) { } // Unrecognized encoding name
             }
 
+            HashData hashData = null;
+            AnalysisTargetToHashDataMap?.TryGetValue(fileLocation.Uri.LocalPath, out hashData);
```

Comment:
>hashData [](http://example.com/codeflow?start=85&length=8)

You should be able to just provide an underscore here, to indicate retrieval of an unused thing. Then you don't need the unused local.

Reply:
since AnalysisTargetToHashDataMap can be null, I must initialize HashData variable  which will be used in the next call.